### PR TITLE
Changelog 24.11.0

### PR DESCRIPTION
**Pull request by beeankha**
## Description

✂️ snip snip ✂️ the making of a new release.

Xref https://github.com/conda/conda-build/issues/5524

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+
+### Bug fixes
+
+* Fixed regex for Jinja2 `set` / `for` statements to be more specific. (#5514)
+* Fixed ruamel.yaml usage to use supported APIs. (#5517)
+* Fixed bug in docs build by pinning conda-sphinx-theme to 0.2.2. (#5518)
+* Fixed bug variant variables were not defined for the first parsing pass of a recipe. (#5528)
+* Fixed a bug where variants were incorrectly found as being used when they matched a leading substring of
+  another variant. (#5535)
+* Fixed a bug where variants were not found when variables were used in ``pin_*`` statements. (#5535)
```
Comment (beeankha) [inline CHANGELOG.md:21]:
```suggestion
* Fix regex for Jinja2 `set` / `for` statements to be more specific. (#5514)
* Fix `ruamel.yaml` usage to use supported APIs. (#5517)
* Fix bug in docs build by pinning `conda-sphinx-theme` version to 0.2.2. (#5518)
* Fix bug variant variables were not defined for the first parsing pass of a recipe. (#5528)
* Fix a bug where variants were incorrectly found as being used when they matched a leading substring of
  another variant. (#5535)
* Fix a bug where variants were not found when variables were used in `pin_*` statements. (#5535)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
 [//]: # (current developments)
 
+## 24.11.0 (2024-11-15)
+
+### Enhancements
+
+* `--package-format` introduced as command line argument. (#4890 via #5209)
```
Comment (beeankha) [inline CHANGELOG.md:7]:
```suggestion
* Introduce `--package-format` as command line argument. (#4890 via #5209)
```

File: .authors.yml [hunk 3]
```diff
@@ -1251,7 +1251,7 @@
   github: beeankha
   alternate_emails:
   - beeankha@gmail.com
-  num_commits: 43
+  num_commits: 39
```
Comment (beeankha) [inline .authors.yml:1254]: Not sure why the number of commits got lower here... 🤔

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
 [//]: # (current developments)
 
+## 24.11.0 (2024-11-15)
+
+### Enhancements
+
+* Introduce `--package-format` as command line argument. (#4890 via #5209)
```
Comment (beeankha) [inline CHANGELOG.md:7]:
```suggestion
* Introduce `--package-format` as a command line argument. (#4890 via #5209)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,54 @@
...
 
+## 24.11.0 (2024-11-15)
+
+### Enhancements
+
+* Introduce `--package-format` as a command line argument. (#4890 via #5209)
+  * This takes precedence over default value and `condarc`.
+  * Normalization occurs so `1`, `"1"`, `tar.bz2`,`.tar.bz2`, `2`, `"2"`, `conda`, `.conda` are all recognized and mapped appropriately.
+  * Other options are rejected.
+* Add  support for CEP-17 that allows specifying the location of the site-packages directory with the `python_site_packages_path` build option for any packages named `python`. (#5502)
```
Comment (beeankha) [inline CHANGELOG.md:11]:
```suggestion
* Add  support for [CEP-17](https://github.com/conda/ceps/blob/main/cep-0017.md) that allows specifying the location of the site-packages directory with the `python_site_packages_path` build option for any packages named `python`. (#5502)
```

File: .authors.yml (hunk 3 above)
Comment (beckermr) [reply .authors.yml:1254]: I'd bump it right back up!

**Review by beckermr — APPROVED** [review]
LGTM! We may want to wait until Monday to merge to avoid breaking things over the weekend.

**Comment by beeankha** [comment]
@beckermr thank you for the review! Agreed, I'd like to wait to merge this (+ tag the 24.11.0 release) until next week.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,54 @@
...
+  * This takes precedence over default value and `condarc`.
+  * Normalization occurs so `1`, `"1"`, `tar.bz2`,`.tar.bz2`, `2`, `"2"`, `conda`, `.conda` are all recognized and mapped appropriately.
+  * Other options are rejected.
+* Add  support for [CEP-17](https://github.com/conda/ceps/blob/main/cep-0017.md) that allows specifying the location of the site-packages directory with the `python_site_packages_path` build option for any packages named `python`. (#5502)
+
+### Bug fixes
+
+* Fix regex for Jinja2 `set` / `for` statements to be more specific. (#5514)
+* Fix `ruamel.yaml` usage to use supported APIs. (#5517)
+* Fix bug in docs build by pinning `conda-sphinx-theme` version to 0.2.2. (#5518)
```
Comment (jaimergp) [inline CHANGELOG.md:17]: Does this belong to `# Docs` better?

**Review by jaimergp — APPROVED** [review]

**Review by jezdez — APPROVED** [review]

File: CHANGELOG.md (hunk 6 above)
Comment (jezdez) [reply CHANGELOG.md]: There is a bug in rever that miscounts things, was fixed in https://github.com/regro/rever/pull/266 but that was never released, so you likely used a version that miscounts contributions basically

Comment (jezdez) [reply CHANGELOG.md]: You can redo the run if you want with that new version

File: .authors.yml (hunk 3 above)
Comment (kenodegard) [reply .authors.yml:1254]: The commit counts are weird for whomever last did the release due to our commit squashing policy

**Pull request merged**